docs/attestations/slsa-provenance.md

```diff
@@ -51,13 +51,14 @@ and exported with your build result, in the root directory.
 
 ## Parameters
 
-| Parameter      | Type           | Default           | Description                                                                                                 |
-| -------------- | -------------- | ---------------- | ----------------------------------------------------------------------------------------------------------- |
-| `mode`         | `min`,`max`    | `max`             | Configures the amount of provenance to be generated. See [mode](#mode)                                      |
-| `builder-id`   | String         |                   | Explicitly set SLSA [`builder.id`](https://slsa.dev/provenance/v0.2#builder.id) field                       |
-| `filename`     | String         | `provenance.json` | Set filename for provenance attestation when exported with `local` or `tar` exporter |
-| `reproducible` | `true`,`false` | `false`           | Explicitly set SLSA [`metadata.reproducible`](https://slsa.dev/provenance/v0.2#metadata.reproducible) field |
-| `inline-only`  | `true`,`false` | `false`           | Only embed provenance into exporters that support inline content. See [inline-only](#inline-only)           |
+| Parameter      | Type           | Default           | Description                                                                                       |
+|----------------|----------------|-------------------|---------------------------------------------------------------------------------------------------|
+| `mode`         | `min`,`max`    | `max`             | Configures the amount of provenance to be generated. See [mode](#mode)                            |
+| `builder-id`   | String         |                   | Explicitly set SLSA Builder ID field. See [builder-id](#builder-id)                               |
+| `filename`     | String         | `provenance.json` | Set filename for provenance attestation when exported with `local` or `tar` exporter              |
+| `reproducible` | `true`,`false` | `false`           | Explicitly marked as reproducible. See [reproducible](#reproducible)                              |
+| `inline-only`  | `true`,`false` | `false`           | Only embed provenance into exporters that support inline content. See [inline-only](#inline-only) |
+| `version`      | String         | `v0.2`            | SLSA provenance version to use (`v0.2` or `v1`)                                                   |
```

Comment:
Any reason to still stick to v0.2 by default?

Reply:
For backwards compatibility with existing tools. We need to give the tools some time to upgrade their compatibility and test, then we can switch the default.